providers/default.rb

```diff
@@ -79,10 +74,6 @@ def initialize(new_resource, run_context)
 
 private
 
-def zk_dependency_gems
-  %w(zookeeper json).collect { |gem| Chef::ResourceResolver.resolve(:chef_gem).new(gem, @run_context) }
```

Comment:
I would keep `zk_dependency_gems` resource but remove `zookeeper` and `json` from the list